backend/remote-state/azure/helpers_test.go

```diff
@@ -156,10 +156,7 @@ func (c *ArmClient) buildTestResources(ctx context.Context, names *resourceNames
 		Location: &names.location,
 	}
 	if names.useAzureADAuth {
-		allowSharedKeyAccess := false
-		storageProps.AccountPropertiesCreateParameters = &armStorage.AccountPropertiesCreateParameters{
-			AllowSharedKeyAccess: &allowSharedKeyAccess,
-		}
+		storageProps.AccountPropertiesCreateParameters = &armStorage.AccountPropertiesCreateParameters{}
```

Comment:
I just remove this field, as it's not available in the older API profile. It doesn't seem to affect whether the test passes.